src/main/resources/containers.json

```diff
@@ -1250,8 +1250,6 @@
     ]
   },
   {
-    "artifact_id": "glassfish",
-    "group_id": "org.glassfish.main.distributions",
     "name": "Arquillian Container GlassFish Managed",
```

Comment:
I believe we don't provide configuration option in chameleon like bindHttpPort, bindAddress for any container which we are providing in forge. So We should have some identifier in this configuration which we'll use to map this configuration properties to respective container.